src/identities/dto/rotate-primary-key-params.dto.ts

```diff
@@ -0,0 +1,24 @@
+/* istanbul ignore file */
+
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { IsDate, IsOptional, IsString } from 'class-validator';
+
+import { TransactionBaseDto } from '~/common/dto/transaction-base-dto';
+
+export class RotatePrimaryKeyParamsDto extends TransactionBaseDto {
+  @ApiProperty({
+    description: 'Account address to which to rotate the primary key',
```

Comment:
```suggestion
    description: 'Account address to which to will become the primary key',
```

IMO just "rotate" is a bit ambiguous and is already in the name